src/easier68k/core/opcodes/dc.py

```diff
@@ -0,0 +1,392 @@
+
+        :param command: The command itself (e.g. 'MOVE.B', 'LEA', etc.)
+        :param parameters: The parameters after the command (such as the source and destination of a move)
+        """
+        valid, issues = DC.is_valid(command, parameters)
+        assert valid, 'Invalid command'
+        # We're good without asserts from here on out
+
+        parts = command.split('.')  # Split the command by period to get the size of the command
+        if len(parts) == 1:  # Use the default size
+            size = 'W'
+        else:
+            size = parts[1].upper()
+
+        params = []
+        quote_delim = None  # If None, we're not in a quote: if it has a value, we know what we're looking for to break the quote
+        current_param = ''
+        is_string = False
+
+        for c in parameters:
+            if c == ',' and not quote_delim:  # End of this parameter (and not a comma in a quote)
+                if current_param:
+                    if is_string:
+                        temp_params = []
+                        for char in current_param:
+                            temp_params.append('$' + format(ord(char), "x"))
+
+                        # Right pad the string with zeroes
+                        if size == 'L':
+                            while len(temp_params) % 4 != 0:
+                                temp_params.append('$00')
+                        if size == 'W':
+                            while len(temp_params) % 2 != 0:
+                                temp_params.append('$00')
+
+                        params.extend(temp_params)
+                    else:
+                        # Make the value the right hex length
+                        val = parse_literal(current_param.strip())
+                        hexed = hex(val)[2:].upper()
+                        if size == 'L':
+                            while len(hexed) % 8 != 0:
+                                hexed = '0' + hexed
+
+                        if size == 'W':
+                            while len(hexed) % 4 != 0:
+                                hexed = '0' + hexed
+
+                        if size == 'B':
+                            while len(hexed) % 2 != 0:
+                                hexed = '0' + hexed
+
+                        # Length is now for sure even
+                        for i in range(0, len(hexed), 2):
+                            params.append('$' + hexed[i:i+2])
+
+                is_string = False
+                quote_delim = None
+                current_param = ''
+                continue
+
+            if quote_delim and c == quote_delim:  # Breaking out of the quote
+                quote_delim = None
+                continue
+
+            if c == '\'':
+                if not quote_delim:  # This is the start of a quote
+                    quote_delim = '\''
+                    current_param = ''
+                    is_string = True
+                    continue
+
+            if c == '\"':
```

Comment:
I don't think so personally (especially since this is being changed drastically anyways so now we'll only have one type of quote to look for), I feel like for how little this is used it's fine as just being hard-coded (but if you think it's really a big deal I can change it)